utils/automation/FoScanner/CliOptions.py

```diff
@@ -40,6 +40,7 @@ class CliOptions(object):
   keyword: bool = False
   repo: bool = False
   diff_dir: str = os.getcwd()
+  keyword_conf_file_path : str = ''
```

Comment:
Please document the class variable in the class documentation as `:ivar:`

Reply:
Corrected!